src/main/java/com/microsoft/cse/helium/app/Constants.java

```diff
@@ -15,4 +15,7 @@
   public static final Integer MAX_PAGE_COUNT = 1000;
   public static final String WEB_INSTANCE_ROLE = "WEBSITE_ROLE_INSTANCE_ID";
   public static final Integer MAX_DEGREE_PARALLELISM = 2;
+
```

Comment:
Let's be consistent for static final and choose one format.  We have caps and underscores for the first three and pascal casing for the last two.

Reply:
Done!